### DISCUSSION TITLE
aichat插件的正确使用方法是什么啊？

### DISCUSSION THREAD

**Discussion by AAAflowerseller** [Future-Scholars/paperlib#555] [Q&A]
![image](https://github.com/Future-Scholars/paperlib/assets/116947227/65521eb6-37a1-4f1c-b1d4-c95dbfe6cb9a)
![image](https://github.com/Future-Scholars/paperlib/assets/116947227/ddeb1e15-336e-4c77-a94c-cd182b165618)
如图，我使用的是gemini，梯子也连了，为什么还是wrong？而且llama解析pdf的时间也很长sad

**Comment by AAAflowerseller** [comment]
![image](https://github.com/Future-Scholars/paperlib/assets/116947227/02b058cb-9cf6-4019-a815-f3c3a70badc8)

**Reply by GeoffreyChen777** [reply]
首先你要确定你的地区可以用 gemini。你可以用ai-summary插件，也选成gemini，试一试。其次你试试用英文问问题。

至于llama解析pdf时间这个是有很多因素，首先你上传pdf给解析api的网速，你pdf文件的大小，以及llama解析本身需要的计算时间。你大概需要多久？

**Reply by AAAflowerseller** [reply]
ai-summary运行成功，但是结果是空白。ai-chat没改配置，今天又试了一次，直接超时。梯子是us的。可以集成chatglm吗？

**Reply by GeoffreyChen777** [reply]
要不加个微信方便沟通一些?
https://share.1password.com/s#1nSLUMJUQ98veCHJvzys4TMBu04b0W964N5DY9hPbSU

**Reply by AAAflowerseller** [reply]
谢谢大佬，我今天尝试用chatglm成功啦！不好意思我不常看消息，前阵子在忙论文，私密马赛orz
